DESCRIPTION

```diff
@@ -34,6 +34,7 @@ Imports:
     lifecycle (>= 0.2.0),
     shinyvalidate,
     teal.code (>= 0.4.1.9009),
+    teal.data (>= 0.3.0.9018),
```

Comment:
It is used in the codebase

https://github.com/search?q=repo%3Ainsightsengineering%2Fteal.osprey%20teal.data%3A%3A&type=code